chains/solana/contracts/tests/testutils/wrapped.go

```diff
@@ -0,0 +1,83 @@
+package testutils
+
+import (
+	"context"
+	"testing"
+
+	"github.com/gagliardetto/solana-go"
+	"github.com/gagliardetto/solana-go/rpc"
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink-ccip/chains/solana/gobindings/ccip_router"
+	"github.com/smartcontractkit/chainlink-ccip/chains/solana/utils/ccip"
+	"github.com/smartcontractkit/chainlink-ccip/chains/solana/utils/common"
+)
+
+// this files includes wrapped methods to be used in testing without additional error checks
```

Comment:
I'm not sure if I follow why this file is named `wrapped` when it's all related to transactions and instructions, right?

Reply:
it's mostly just the actual implementation is elsewhere and these are test wrappers - i'm open to more suggested names if someone has a preference